src/test/osd/RadosModel.h

```diff
@@ -2729,10 +2909,14 @@ class TierFlushOp : public TestOp {
     cout << num << ":  got " << cpp_strerror(r) << std::endl;
     if (r == 0) {
       // sucess
+    } else if (r == -EBUSY) {
+      // could fail if snap is not oldest
```

Comment:
Can't we check this?

Reply:
Hm.. How can we check snap is not the oldest because snap is selected randomly?

Reply:
How hard would it be to track the clones that should exist for each object?  I suppose this might be out of scope, but it definitely weakens the test.